engine/table/src/main/java/io/deephaven/engine/exceptions/ColumnSnapshotUnsuccessfulException.java

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.exceptions;
+
+import io.deephaven.UncheckedDeephavenException;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This exception is thrown when {@link io.deephaven.engine.table.impl.remote.ConstructSnapshot} fails to successfully
+ * collect snapshot data for a column in parallel.
+ */
+public class ColumnSnapshotUnsuccessfulException extends UncheckedDeephavenException {
```

Comment:
I did mean `ColumnSnapshot...` because this exception is thrown when parallel snapshot for a column fails.
There is another `SnapshotUnsuccessfulException` which is more generic.